gooddata-java/src/main/java/com/gooddata/sdk/service/project/ProjectService.java

```diff
@@ -124,6 +124,14 @@ public ProjectService(final RestTemplate restTemplate, final AccountService acco
         return new PageBrowser<>(startPage, page -> listProjects(getProjectsUri(userId, page)));
     }
 
+    public PageBrowser<Project> listProjects(final Account account, final PageRequest startPage) {
+        notNull(startPage, "startPage");
+        notNull(account, "account");
+        notEmpty(account.getId(), "account.uri");
+        final String userId = account.getId();
```

Comment:
It's not necessary to create new variable when it's obvious, so you can inline this ;)